examples/simple/src/comments/CommentEdit.js

```diff
@@ -1,6 +1,6 @@
 import Card from '@material-ui/core/Card';
 import Typography from '@material-ui/core/Typography';
-import { withStyles } from '@material-ui/core/styles';
+import { makeStyles } from '@material-ui/styles';
```

Comment:
Please use `makeStyles` from `@material-ui/core/styles` (here an in other imports). I don't remember why, but it's the one we should use.